doc/badge.md

```diff
@@ -0,0 +1,87 @@
+# Badge Component - Default variant
+
+### What is it?
+Badge represents dynamic information such as number of notifications unread with some styles to match Nat DS appearance.
+
+### Why should I use it?
+This component can help you to show some simple information on the top of chosen image.
+
+### When should I use it?
+Every time you want to display an image with a badge
```

Comment:
Aqui eu descreveria de forma mais genérica: "Every time you want to display _a component_ with a badge"

Reply:
huhum. ajustado.